docs/src/main/asciidoc/funqy-http.adoc

```diff
@@ -11,6 +11,13 @@ include::./attributes.adoc[]
 The guide walks through quickstart code to show you how you can deploy Funqy as a
 standalone service and invoke on Funqy functions using HTTP.
 
+WARNING: The Funqy HTTP binding is not a replacement for REST over HTTP.  Because Funqy
+needs to be portable cross a lot of different protocols and function providers its HTTP binding
+is very minimalistic and you will lose REST features like linking and the ability to leverage
+HTTP features like cache-control and conditional GETs.  You may want to consider using Quarkus's
+JAX-RS, Spring MVC, or Vert.x Web Reactive Route support instead, although Funqy will have less overhead
+than these alternatives other than Vert.x.
```

Comment:
Maybe include a pointer to the relevant guide?

I also think we need to try and explain conceptually the difference between a function and a HTTP Endpoint, although I am not really sure how to word this.

Reply:
IMO, function is just a buzzword.  They are all endpoints.

Reply:
Kinda, but they are single purpose endpoints. HTTP has a path, a method and headers, so the one deployment will generally provide a few different function (e.g. even a simple CRUD endpoint for a single entity will have different methods for GET, PUT, DELETE etc). IMHO difference between a function and server-less HTTP is that a function only done one thing. If you want to do CRUD with functions you need 4 different functions, one for each operation, while for HTTP you have a single endpoint that handles different methods.

Reply:
Yeah, people love to talk about semantics, but in the end, most devs just use all of this for RPC.

Reply:
added some links